src/pages/studyView/table/FixedHeaderTable.tsx

```diff
@@ -364,26 +364,28 @@ export default class FixedHeaderTable<T> extends React.Component<
         const showDeselectAll = !noneSelected && this.props.removeAll;
 
         return (
-            <>
+            <div style={{ overflow: 'auto', width: '50%' }}>
```

Comment:
what's reason for this 50%?    i would like to see the input on right side.  you can achieve this with flexbox styles